site/en/getstarted/prerequisite-helm.md

```diff
@@ -11,28 +11,23 @@ summary: Learn the necessary preparations before installing Milvus.
 
 Before you install Milvus, check your hardware and software to see if they meet the requirements.
 
-{{tab}}
```

Comment:
Why are we removing this? This syntax renders as below:
![image](https://user-images.githubusercontent.com/83751381/177909329-d2e46a69-21f8-4f96-89fa-9943ac0b9227.png)

Reply:
This mistake has been rectified along with other errors and clarity has been improved. Disk requirements have also been added in prerequisite-docker.md.